index.html

```diff
@@ -15,24 +15,24 @@
         <input disabled type="text" id="display" value="0" name="display" class="calc-display-input"/>
       </div>
       <div class="calc__button">
-        <button class="clear" id="c">c</button>
-        <button class="clear" id="ce">ce</button>
-        <button class="operation-style operation">/</button>
-        <button class="operation-style operation">*</button>
-        <button class="number-style number">7</button>
-        <button class="number-style number">8</button>
-        <button class="number-style number">9</button>
-        <button class="operation-style operation">-</button>
-        <button class="number-style number">4</button>
-        <button class="number-style number">5</button>
-        <button class="number-style number">6</button>
-        <button class="operation-style operation">+</button>
-        <button class="number-style number">1</button>
-        <button class="number-style number">2</button>
-        <button class="number-style number">3</button>
-        <button class="button-zero number-style number">0</button>
-        <button class="number-style" id="dot">.</button>
-        <button class="button-equals operation-style operation">=</button>
+        <button class="button_clear" id="c">c</button>
```

Comment:
The class name doesn't conform with BEM. Please, give the BEM-class-name.

Reply:
DONE!